web_src/css/repo.css

```diff
@@ -654,13 +654,16 @@ td .commit-summary {
 
 .repository.view.issue .comment-list .code-comment {
   border: 1px solid transparent;
-  margin: 0;
+  margin: 0 -4px;
+  padding: 8px;
```

Comment:
The spaces are provided by `.repository.view.issue .comment-list .ui.comments`, so I think we should avoid the new negative margins.

Reply:
I think it's a good time to clean up legacy style conflicts, then it will be easier to make more UI improvements.

And we don't need to fully backport these changes.